core/templates/dev/head/collection_editor/CollectionStatistics.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controllers and services for the collection statistics in the
+ * collection editor page.
+ *
+ * @author oskar.cieslik@gmail.com (Oskar Cieslik)
+ */
+
+oppia.controller('CollectionStatistics', ['$scope']);
```

Comment:
No need to inject ``$scope`` here, since it isn't used yet. You can pass an empty array, instead. Ditto for most of the other new controllers, too.